mysql-test/suite/rocksdb/t/manual_compaction_bottommost_level-master.opt

```diff
@@ -0,0 +1,3 @@
+--rocksdb_max_subcompactions=1
+--rocksdb_default_cf_options=target_file_size_base=100k;max_bytes_for_level_multiplier=1;max_bytes_for_level_base=1m;target_file_size_multiplier=1
```

Comment:
`--rocksdb_` should be always `--loose_rocksdb_` in case RocksDB is not compiled